packages/genesys-spark-components/src/components/beta/gux-tooltip-base/gux-tooltip-base.tsx

```diff
@@ -156,17 +167,21 @@ export class GuxTooltipBase {
     });
   }
   private show(): void {
+    clearTimeout(this.hideDelayTimeout);
     this.isShown = true;
     afterNextRender(() => {
       this.runUpdatePosition();
     });
   }
 
   private hide(): void {
-    if (this.cleanupUpdatePosition) {
-      this.cleanupUpdatePosition();
-    }
-    this.isShown = false;
+    this.hideDelayTimeout = setTimeout(() => {
+      this.isShown = false;
+
+      if (this.cleanupUpdatePosition) {
+        this.cleanupUpdatePosition();
+      }
+    }, 350);
```

Comment:
Is there a reason it's 350? Is it specified? Just curious

Reply:
@jordanstith15  https://inindca.atlassian.net/browse/COMUI-3128